pkg/common/utils/utils.go

```diff
@@ -274,8 +275,9 @@ func QueryAllVolumesForCluster(ctx context.Context, m cnsvolume.Manager, cluster
 }
 
 func GetVirtualMachineAllApiVersions(ctx context.Context, vmKey types.NamespacedName,
```

Comment:
I think we can obtain API version from the VM. Why we need to add one more return value for apiVersion?